app/components/amount-input-screen/number-pad-reducer.ts

```diff
@@ -54,6 +61,8 @@ export const Key = {
 
 export type Key = (typeof Key)[keyof typeof Key]
 
+export type Keys = `${number | ""}${"." | ""}${number | ""}`
```

Comment:
assuming Keys is what the user can paste, I don't think it make sense really a type. because ultimately the user could paste anything: a `string`, a `number`. I think we will want to validate / trim the input to make sure that if the user paste something incorrect, this is correctly handled.

Reply:
said another way, I don't think we need a `Keys` type at all, and we can accept a `string` instead that should be tentatively converted to a number.

Reply:
Yea i agree the Keys type seemed unnecessary. Removing that would streamline the code here too by removing unnecessary type conversions. Ill adjust the whole process to accept a string.

The safety and validation is already there too. Note the regex matching on line 68 in amount-input-screen-ui.tsx.